tests/sync/assessmentsSync.js

```diff
@@ -485,10 +485,34 @@ describe('Assessment syncing', () => {
     courseData.courseInstances[util.COURSE_INSTANCE_ID].assessments['fail'] = assessment;
     await util.writeAndSyncCourseData(courseData);
     const syncedAssessment = await findSyncedAssessment('fail');
-    assert.match(syncedAssessment.sync_errors, /Cannot specify an array of points for a question/);
+    assert.match(syncedAssessment.sync_errors, /Cannot specify more than 1 point value for a question/);
   });
 
-  it('records an error if points array is specified for an alternative when real-time grading is disallowed', async () => {
+  it('accepts one-element points array being specified for a question when real-time grading is disallowed', async () => {
+    const courseData = util.getCourseData();
+    const assessment = makeAssessment(courseData);
+    assessment.allowRealTimeGrading = false;
+    assessment.zones = [{
+      title: 'zone 1',
+      questions: [{
+        id: util.QUESTION_ID,
+        points: [5],
+      }, {
+        id: util.ALTERNATIVE_QUESTION_ID,
+        points: [10],
+      }],
+    }];
+    courseData.courseInstances[util.COURSE_INSTANCE_ID].assessments['points_array_size_one'] = assessment;
+    await util.writeAndSyncCourseData(courseData);
+    const syncedAssessment = await getSyncedAssessmentData('points_array_size_one');
```

Comment:
Maybe we should explicitly check that there weren't any sync errors for this assessment, like this: https://github.com/PrairieLearn/PrairieLearn/blob/400677ab19fa544354138862ab00124369b52d3d/tests/sync/assessmentsSync.js#L646